cfme/tests/cli/test_appliance_log_rotate.py

```diff
@@ -0,0 +1,44 @@
+import datetime
+
+import dateutil
+import pytest
+
+LOGS = ["/var/www/miq/vmdb/log/evm.log",
```

Comment:
Please consider comparing both files for one test run. Your test requests new appliance for every log file,  it doesn't look efficient to me